### PR TITLE
Remove failing scheduling tests

### LINKED ISSUE #73: Update scheduling related tests
#### What would you like to be added?

- As per [KEP-2578: Windows Operational Readiness Specification](https://github.com/knabben/enhancements/blob/3ee6784895d650caa99e3ad23a361465478effb5/keps/sig-windows/2578-windows-conformance/README.md), the project needs to include `Core.Scheduling` tests to ensure that a cluster can handle pass basic scheduling related tests
- As such, basic scheduling tests need to be updated to ensure that they are conforming to the KEP specifications
- Basic scheduling tests were last updated in February 2022, and the following updates may be necessary
  - Update and identify new basic scheduling test case candidates and ensure KEP is updated to reflect this
  - Ensure basic scheduling tests are successfully being executed successfully and where necessary, update upstream kubernetes e2e tests to ensure tests execute successfully
  - Only 2 commented out test cases are currently part of the test suite. Consider uncommenting these tests 
#### Why is this needed?

- Refer to [KEP-2578: Windows Operational Readiness Specification](https://github.com/knabben/enhancements/blob/3ee6784895d650caa99e3ad23a361465478effb5/keps/sig-windows/2578-windows-conformance/README.md) for motivation for the need for basic scheduling tests as part of the Windows Operational Readiness Conformance tests

### PR DESCRIPTION
**Pull request by tzifudzi**
## What type of PR is this?
New enhancement

## What this PR does / why we need it:
**TLDR: Removes 3 test cases to be ready for initial ops readiness release**

This PR is part of an effort to achieve a minimal set of test cases that will be part of an initial release. As part of this effort, `sig-windows` team is going category by category and finalizing on which test cases are appropriate for the initial release.

I removed three commented-out test cases. 

#### Irrelevant test cases removed
Two of three of the test cases, were more suited to verifying the behaviour of the Kubernetes scheduler, which is a control plane component, rather than specifically testing Windows nodes within the data plane. These can be removed.
1. Ability to preempt a lower priority pod in favor of scheduling higher priority pod
2. Ability to demonstrate the pods requesting more CPU and Memory then available are left in the Pending state over time

#### Relevant test cases removed
One of the three test cases was well suited for testing against a Windows node, but during my tests on AWS EKS, the test was failing because of a specific manner in which the test is written, than the fact the Windows node is failing the tests. More specifically, the test is failing with the following log.
```
2023-12-05T00:41:48-08:00       INFO    [sig-scheduling] LimitRange [It] should create a LimitRange with defaults and ensure pod has those defaults applied. [Conformance]
...
2023-12-05T00:41:48-08:00       INFO      [FAILED] resource [vpc.amazonaws.com/PrivateIPv4Address](http://vpc.amazonaws.com/PrivateIPv4Address) expected 0 actual 1
2023-12-05T00:41:48-08:00       INFO      In [It] at: test/e2e/scheduling/limit_range.go:137 @ 12/05/23 00:41:47.876
```

The reason for the failure is that the [AWS EKS VPC Resource Controller](https://github.com/aws/amazon-vpc-resource-controller-k8s) updates a field that the test case does not expect to be updated. This field has to be updated in order for AWS EKS IP address management to work as expected and is by design. Added an issue https://github.com/kubernetes-sigs/windows-operational-readiness/issues/101 to track this as technical debt for a future ops readiness release. I am currently testing op-readiness using AWS EKS and therefore have no other means of verifying this functionality and the issue will have to be addressed at a later time.

## Which issue(s) this PR fixes:
Resolves https://github.com/kubernetes-sigs/windows-operational-readiness/issues/73 to finish updating scheduling tests to be ready for the initial release of ops-readines

### CODE REVIEW

**Comment by knabben** [comment]
Can them be moved to an issue as TODO, or be cleaned up from the KEP?

**Comment by tzifudzi** [comment]
@knabben I did create a separate issue to track https://github.com/kubernetes-sigs/windows-operational-readiness/issues/101. Will cross-check if we need to update the KEP. 👍🏾
